pipeline/test/test_pipeline.py

```diff
@@ -245,7 +245,8 @@ def test_add_extension_input_resource_file(self):
 
 class BatchTests(unittest.TestCase):
     def pipeline(self):
-        return Pipeline(backend=BatchBackend(os.environ.get('BATCH_URL')),
+        return Pipeline(backend=BatchBackend(os.environ.get('BATCH_URL'),
+                                             user='pipeline-test-0-1--hail-is'),
```

Comment:
Formatting!

Reply:
I think this is actually right, `user` is a keyword argument to `BatchBackend` and the close-paren ends the `BatchBackend` constructor.

Reply:
ok.